webkit/0001-Add-test-runner-for-the-source-map-spec-tests.patch

```diff
@@ -2591,7 +2591,7 @@ index 000000000000..c5fbd4baa8b0
 ++++ b/front_end/core/sdk/fixtures/sourcemaps/invalid-mapping-segment-with-three-fields.js
```

Comment:
Wow GitHub makes it very hard to review diffs of patch files

Reply:
oh yeah it's not ideal 🥲

perhaps there's some `linguist-` we could add to `.gitattributes` that would make it slightly nicer?